appliances/Community/Lithops/appliance.sh

```diff
@@ -0,0 +1,255 @@
+    'ONEAPP_MINIO_SECRET_ACCESS_KEY'    'configure'  'Lithops storage backend MinIO account user secret access key'     'O|text'
+    'ONEAPP_MINIO_BUCKETT'              'configure'  'Lithops storage backend MinIO existing bucket'                    'O|text'
+    'ONEAPP_MINIO_ENDPOINT_CERT'        'configure'  'Lithops storage backend MinIO endpoint certificate'               'O|text64'
+)
+
+
+### Appliance metadata ###############################################
+
+# Appliance metadata
+ONE_SERVICE_NAME='Service Lithops - KVM'
+ONE_SERVICE_VERSION='3.4.0'   #latest
+ONE_SERVICE_BUILD=$(date +%s)
+ONE_SERVICE_SHORT_DESCRIPTION='Appliance with preinstalled Lithops for KVM hosts'
+ONE_SERVICE_DESCRIPTION=$(cat <<EOF
+Appliance with preinstalled Lithops v3.4.0.
+
+By default, it uses localhost both for Compute and Storage Backend.
+
+To configure MinIO as Storage Backend use the parameter ONEAPP_STORAGE=minio
+with ONEAPP_MINIO_ENDPOINT, ONEAPP_MINIO_ACCESS_KEY_ID and ONEAPP_MINIO_SECRET_ACCESS_KEY.
+These parameters values have to point to a valid and reachable MinIO server endpoint.
+
+The parameter ONEAPP_MINIO_BUCKETT and ONEAPP_MINIO_ENDPOINT_CERT are optional.
+- ONEAPP_MINIO_BUCKET points to an existing bucket in the MinIO server. If the bucket does not exist or if the
+parameter is empty, the MinIO server will generate a bucket automatically.
+- ONEAPP_MINIO_ENDPOINT_CERT is necessary when using self-signed certificates on the MinIO server. This is the
+certificate for the CA on the MinIO server. If the CA certificate exists, script will skip it,
+if one would want to update the CA certificate from context, first delete previous ca.crt file.
+EOF
+)
+ONE_SERVICE_RECONFIGURABLE=true
+
+### Contextualization defaults #######################################
+
+ONEAPP_BACKEND="${ONEAPP_BACKEND:-localhost}"
+ONEAPP_STORAGE="${ONEAPP_STORAGE:-localhost}"
+
+### Globals ##########################################################
+
+DEP_PKGS="python3-pip"
+DEP_PIP="boto3"
+LITHOPS_VERSION="3.4.0"
+DOCKER_VERSION="5:26.1.3-1~ubuntu.22.04~jammy"
+
+###############################################################################
+###############################################################################
+###############################################################################
+
+#
+# service implementation
+#
+
+service_cleanup()
+{
+    :
+}
+
+service_install()
+{
+    # ensuring that the setup directory exists
+    #TODO: move to service
+    mkdir -p "$ONE_SERVICE_SETUP_DIR"
+    export DEBIAN_FRONTEND=noninteractive
+
+    # packages
+    install_deps ${DEP_PKGS} ${DEP_PIP}
+
+    # docker
+    install_docker
+
+    # Lithops
+    install_lithops
+
+    # create Lithops config file in /etc/lithops
+    create_lithops_config
+
+    # service metadata
+    create_one_service_metadata
+
+    # cleanup
+    postinstall_cleanup
+
+    msg info "INSTALLATION FINISHED"
+
+    return 0
+}
+
+service_configure()
```

Comment:
I'm not sure what exactly gets executed, so docker is installed, some pip packages, but how and where all this is used? :)